### PR TITLE
Reduce DRY limit from 13 to 12 lines

### PR DESCRIPTION
**Pull request by okurz**
https://en.wikipedia.org/wiki/Don%27t_repeat_yourself and https://clean-code-developer.com/grades/grade-1-red/#Don8217t_Repeat_Yourself_DRY

### CODE REVIEW

**Comment by foursixnine** [comment]
@okurz Why?

File: tools/detect_code_dups [hunk 1]
```diff
@@ -76,7 +76,7 @@ sub match_file {
 }
 
 my $verbose             = 1;
-my $minlength           = 13;
+my $minlength           = 11;
```
Comment (asdil12) [inline tools/detect_code_dups]: Either your PR title or this value is wrong.

**Comment by okurz** [comment]
@foursixnine 

> @okurz Why?

-> https://en.wikipedia.org/wiki/Don%27t_repeat_yourself and https://clean-code-developer.com/grades/grade-1-red/#Don8217t_Repeat_Yourself_DRY

**Comment by foursixnine** [comment]
@okurz my comment comes more in the form of: What's the reasoning to do the change, as you're one of the main persons asking for explanations in PR's and commits.

**Comment by okurz** [comment]
ok. I have put that in the PR description now

**Pull request closed without merging**